// MIT License

// Copyright (c) 2020 jianlinjiang

syntax = "proto3";
option cc_generic_services = true;

package ra;

message RaRequest{
  string message = 1;
};

message RaResponse {
  string message = 1;
};

service RaService {
  rpc Ra(RaRequest) returns (RaResponse);
}